--- mlt-6.0.0/src/modules/core/transition_matte.c~	2016-02-18 00:43:24.000000000 +0100
+++ mlt-6.0.0/src/modules/core/transition_matte.c	2016-02-27 18:56:14.775267267 +0100
@@ -41,7 +41,7 @@
 	(
 		"movdqu         (%[equ255]), %%xmm4     \n\t"   /* load bottom value 0xff */
 
-		"loop_start1:                           \n\t"
+		"matte_loop_start1:                      \n\t"
 
 		/* load pixels block 1 */
 		"movdqu         0(%[image_b]), %%xmm0   \n\t"
@@ -64,7 +64,7 @@
 
 		/* loop if we done */
 		"dec            %[cnt]                  \n\t"
-		"jnz            loop_start1             \n\t"
+		"jnz            matte_loop_start1       \n\t"
 		:
 		: [cnt]"r" (cnt), [alpha_a]"r"(alpha_a), [image_b]"r"(image_b), [equ255]"r"(const4)
 	);
@@ -119,7 +119,7 @@
 		"movdqu         (%[equ235]), %%xmm5     \n\t"   /* load bottom value 235 */
 		"movdqu         (%[equ255]), %%xmm4     \n\t"   /* load bottom value 0xff */
 
-		"loop_start:                            \n\t"
+		"matte_loop_start:                      \n\t"
 
 		/* load pixels block 1 */
 		"movdqu         0(%[image_b]), %%xmm0   \n\t"
@@ -174,7 +174,7 @@
 
 		/* loop if we done */
 		"dec            %[cnt]                  \n\t"
-		"jnz            loop_start              \n\t"
+		"jnz            matte_loop_start        \n\t"
 		:
 		: [cnt]"r" (cnt), [alpha_a]"r"(alpha_a), [image_b]"r"(image_b), [equ43]"r"(const1), [equ16]"r"(const2), [equ235]"r"(const3), [equ255]"r"(const4)
 	);
