diff -up mlt-6.8.0/src/modules/core/transition_matte.c.0002~ mlt-6.8.0/src/modules/core/transition_matte.c
--- mlt-6.8.0/src/modules/core/transition_matte.c.0002~	2018-05-11 03:16:56.000000000 +0200
+++ mlt-6.8.0/src/modules/core/transition_matte.c	2018-06-23 01:41:52.045927429 +0200
@@ -54,7 +54,7 @@ static void __attribute__((noinline)) co
 		"movdqu         (%[equ235]), %%xmm5     \n\t"   /* load bottom value 235 */
 		"movdqu         (%[equ255]), %%xmm4     \n\t"   /* load bottom value 0xff */
 
-		"loop_start:                            \n\t"
+		"matte_loop_start:                      \n\t"
 
 		/* load pixels block 1 */
 		"movdqu         0(%[image_b]), %%xmm0   \n\t"
@@ -109,7 +109,7 @@ static void __attribute__((noinline)) co
 
 		/* loop if we done */
 		"dec            %[cnt]                  \n\t"
-		"jnz            loop_start              \n\t"
+		"jnz            matte_loop_start        \n\t"
 		:
 		: [cnt]"r" (cnt), [alpha_a]"r"(alpha_a), [image_b]"r"(image_b), [equ43]"r"(const1), [equ16]"r"(const2), [equ235]"r"(const3), [equ255]"r"(const4)
 	);
