diff -Naur mlt-6.20.0/src/modules/opencv/configure mlt-6.20.0.tpg/src/modules/opencv/configure
--- mlt-6.20.0/src/modules/opencv/configure	2020-02-16 22:31:58.000000000 +0000
+++ mlt-6.20.0.tpg/src/modules/opencv/configure	2020-06-05 12:48:22.131812613 +0000
@@ -26,18 +26,18 @@
 		exit 0
 	fi
 
-    pkg-config --atleast-version=3.1.0 'opencv'
+    pkg-config --atleast-version=3.1.0 'opencv4'
 	if [ $? -eq 0 ]
 	then
-                result=`pkg-config --libs opencv | grep "opencv_tracking"`
+                result=`pkg-config --libs opencv4 | grep "opencv_tracking"`
                 if [ -z "$result" ]
                 then
                         echo "- OpenCV tracking contrib module NOT found, disabling OpenCV modules"
                         touch ../disable-opencv
                         exit 0
                 else
-                        echo "CFLAGS += $(pkg-config --cflags opencv)" >> config.mak
-                        echo "LDFLAGS += $(pkg-config --libs opencv)" >> config.mak
+                        echo "CFLAGS += $(pkg-config --cflags opencv4)" >> config.mak
+                        echo "LDFLAGS += $(pkg-config --libs opencv4)" >> config.mak
                 fi
 	else
 	    echo "- OpenCV >= 3.1.0 NOT found: disabling"
