--- mlt-6.4.1/src/modules/opencv/filter_opencv_tracker.cpp.omv~	2018-01-27 20:07:33.200731460 +0100
+++ mlt-6.4.1/src/modules/opencv/filter_opencv_tracker.cpp	2018-01-27 20:14:05.652527635 +0100
@@ -109,13 +109,24 @@ static void analyze( mlt_filter filter,
         {
 		// Build tracker
 		data->algo = mlt_properties_get( filter_properties, "algo" );
-                if ( data->algo == NULL || !strcmp(data->algo, "" ) )
+                if ( data->algo == NULL || !strcmp(data->algo, "" ) || !strcmp(data->algo, "KCF") )
                 {
-			data->tracker = cv::Tracker::create( "KCF" );
-		}
-		else
+			data->tracker = cv::TrackerKCF::create( );
+		} else if(!strcmp(data->algo, "TLD"))
+                {
+			data->tracker = cv::TrackerTLD::create( );
+		} else if(!strcmp(data->algo, "BOOSTING"))
+                {
+			data->tracker = cv::TrackerBoosting::create( );
+		} else if(!strcmp(data->algo, "MEDIAN_FLOW"))
+                {
+			data->tracker = cv::TrackerMedianFlow::create( );
+		} else if(!strcmp(data->algo, "MIL"))
+                {
+			data->tracker = cv::TrackerMIL::create( );
+		} else if(!strcmp(data->algo, "GOTURN"))
                 {
-			data->tracker = cv::Tracker::create( data->algo );
+			data->tracker = cv::TrackerGOTURN::create( );
 		}
 
 		// Discard previous results
