diff -Naur mlt-6.14.0/src/swig/python/build mlt-6.14.0.tpg/src/swig/python/build
--- mlt-6.14.0/src/swig/python/build	2019-03-30 20:59:21.000000000 +0000
+++ mlt-6.14.0.tpg/src/swig/python/build	2019-04-11 14:19:09.280000000 +0000
@@ -13,7 +13,7 @@
 if [ $? = 0 ]
 then
 	# Change this as needed
-	export PYTHON_INCLUDE=`python -c "import sys;print(\"{}/include/python{}.{}\".format(sys.prefix,*sys.version_info))"`
+	export PYTHON_INCLUDE=`python -c "from sysconfig import get_paths; info = get_paths(); print(info['include'])"`
 
 	[ ! -d "$PYTHON_INCLUDE" ] && echo python development missing && exit 1
 
