diff -up mlt-6.20.0/src/modules/qt/filter_qtext.cpp.omv~ mlt-6.20.0/src/modules/qt/filter_qtext.cpp
--- mlt-6.20.0/src/modules/qt/filter_qtext.cpp.omv~	2020-03-24 16:37:31.379066179 +0100
+++ mlt-6.20.0/src/modules/qt/filter_qtext.cpp	2020-03-24 16:37:35.088042778 +0100
@@ -21,6 +21,7 @@
 #include <framework/mlt.h>
 #include <framework/mlt_log.h>
 #include <QPainter>
+#include <QPainterPath>
 #include <QString>
 
 static QRectF get_text_path( QPainterPath* qpath, mlt_properties filter_properties, const char* text, double scale )
diff -up mlt-6.20.0/src/modules/qt/graph.cpp.omv~ mlt-6.20.0/src/modules/qt/graph.cpp
--- mlt-6.20.0/src/modules/qt/graph.cpp.omv~	2020-03-24 16:36:03.570606200 +0100
+++ mlt-6.20.0/src/modules/qt/graph.cpp	2020-03-24 16:36:13.143548703 +0100
@@ -19,6 +19,7 @@
 
 #include "graph.h"
 #include <QVector>
+#include <QPainterPath>
 #include <math.h>
 
 /*
diff -up mlt-6.20.0/src/modules/qt/producer_qtext.cpp.omv~ mlt-6.20.0/src/modules/qt/producer_qtext.cpp
--- mlt-6.20.0/src/modules/qt/producer_qtext.cpp.omv~	2020-03-24 16:37:22.163124125 +0100
+++ mlt-6.20.0/src/modules/qt/producer_qtext.cpp	2020-03-24 16:37:26.544096614 +0100
@@ -26,6 +26,7 @@
 #include <QImage>
 #include <QColor>
 #include <QPainter>
+#include <QPainterPath>
 #include <QFont>
 #include <QString>
 #include <QTextCodec>
