--- mlt-6.4.1/src/framework/mlt_property.h.omv~	2017-08-09 22:28:50.211901393 +0200
+++ mlt-6.4.1/src/framework/mlt_property.h	2017-08-09 22:29:09.612187394 +0200
@@ -30,7 +30,9 @@
 #include <sys/param.h>
 #endif
 
-#if defined(__GLIBC__) || defined(__APPLE__) || (__FreeBSD_version >= 900506)
+#if defined(__GLIBC__)
+#include <locale.h>
+#elif defined(__APPLE__) || (__FreeBSD_version >= 900506)
 #include <xlocale.h>
 #else
 typedef char* locale_t;
